packages/puppeteer/package.json

```diff
@@ -30,9 +30,6 @@
     "type": "git",
     "url": "https://github.com/puppeteer/puppeteer/tree/main/packages/puppeteer"
   },
-  "engines": {
```

Comment:
Why remove? Keep this it prints a warning about you using older version. There was a report about this sometime ago.